From 10e9abc766e4bfe1f8c307024775d913a274429e Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Tue, 14 Jul 2020 19:19:34 +0800
Subject: [PATCH] ARM: dts: rockchip: enable txfifo resize for rv1126 usb

RV1126 EVB USB gadget maybe used as UVC function which
needs to resize txfifo to improve isoc ep throughput,
so this patch adds "snps,tx-fifo-resize" property to
enable txfifo resize.

Change-Id: I00447fb252a093acb24525b12479f67662d9dfe3
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm/boot/dts/rv1126.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 2701042fb711..a3c9c7f856b4 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -2163,6 +2163,7 @@
 			snps,dis_u2_susphy_quirk;
 			snps,dis-del-phy-power-chg-quirk;
 			snps,tx-ipgap-linecheck-dis-quirk;
+			snps,tx-fifo-resize;
 			snps,xhci-trb-ent-quirk;
 			status = "disabled";
 		};
-- 
2.35.3

